src/main/scripts/org/reaktivity/specification/nukleus/http_cache/streams/proxy/edge-arch/polling.403.response.cancels.pending.on-update.requests/connect/server.rpt

```diff
@@ -0,0 +1,70 @@
+#
+# Copyright 2016-2017 The Reaktivity Project
+#
+# The Reaktivity Project licenses this file to you under the Apache License,
+# version 2.0 (the "License"); you may not use this file except in compliance
+# with the License. You may obtain a copy of the License at:
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+
+property newProxyConnectRef ${nuklei:newReferenceId()} # external scope
+property response1Date ${http_cache:date()} # sharedProperty
+property response2Date ${http_cache:datePlus(5)} # sharedProperty
+property etag ${http_cache:strongEtag()} # sharedProperty
+property etag2 ${http_cache:strongEtag()} # sharedProperty
+
+accept await ROUTED_PROXY
+        "nukleus://target/streams/http-cache"
+        option nukleus:route ${newProxyConnectRef}
+        option nukleus:window 8192
+        option nukleus:transmission "half-duplex"
+accepted
+
+read nukleus:begin.ext ${http:header(":scheme", "http")}
+                       ${http:header(":method", "GET")}
+                       ${http:header(":path", "/data")}
+                       ${http:header(":authority", "localhost:8080")}
+
+connected
+
+read closed
+
+write nukleus:begin.ext ${http:header(":status", "200")}
+                        ${http:header("content-length", "11")}
+                        ${http:header("date", response1Date)}
+                        ${http:header("surrogate-control", "max-age=1+100, x-protected")}
+                        ${http:header("etag", etag)}
+
+write "response #1"
+
+write close
+
+accepted
+
+read nukleus:begin.ext ${http:header(":scheme", "http")}
+                       ${http:header(":method", "GET")}
+                       ${http:header(":path", "/data")}
+                       ${http:header(":authority", "localhost:8080")}
+
+connected
+
+read closed
+
+write await REQUEST_TWO_PENDING
+
+write nukleus:begin.ext ${http:header(":status", "403")}
+                        ${http:header("content-length", "11")}
+                        ${http:header("date", response2Date)}
+                        ${http:header("surrogate-control", "max-age=1+100, x-protected")}
+                        ${http:header("etag", etag2)}
+
+write "response #2"
```

Comment:
It seems weird to include what looks like a normal response (body) in a 403 Forbidden response. Maybe have no content, or an error message?